src/AutoRest.CSharp/Common/Input/InputTypes/DurationKnownEncoding.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace AutoRest.CSharp.Common.Input;
+
+internal enum DurationKnownEncoding
+{
+    Iso8601, Seconds, Constant
```

Comment:
Shall we use ISO8601?    And what does constant encoding mean?

Reply:
These values directly come from Typespec: https://github.com/archerzz/typespec/blob/c0376241326e5a8f413736a767dce1e5d4966fd9/packages/compiler/src/lib/decorators.ts#L667-L669

`Constant` is a backward compatible thing we introduced previously using directive.
For details of format in timespan, please refer: https://learn.microsoft.com/en-us/dotnet/api/system.timespan.parseexact?view=net-8.0#system-timespan-parseexact(system-string-system-string-system-iformatprovider-system-globalization-timespanstyles)
Our `constant` means `c` in this document, try search `"c"` (with quotes) and see how it looks like

Reply:
And usually C# uses camel cases when an abbr has 3 or more letters such as `Vmss` therefore I used `Iso` and I think we should change other places that we are using `ISO`.